packages/plugins/src/$AliasPlugin.ts

```diff
@@ -45,21 +50,25 @@ const $AliasPlugin: PluginType<AliasPluginPage> = {
       return;
     }
     for (const fullPath in config.data.aliases) {
-      for (const alias of config.data.aliases[fullPath]) {
-        const aliasPath = path.posix.resolve(path.dirname(fullPath), alias).toLowerCase();
-        const aliasDir = path.dirname(aliasPath);
-        if (!(await mutableFilesystem.promises.exists(aliasDir))) {
-          await mutableFilesystem.promises.mkdir(aliasDir, {
-            recursive: true
-          });
-        }
-        if (!(await mutableFilesystem.promises.exists(aliasPath))) {
-          await mutableFilesystem.promises.symlink(fullPath, aliasPath);
-        } else {
-          console.warn(
-            `Alias '${aliasPath}' already exists. Is there a duplicate alias, or a page with the same name but different file extensions?`
-          );
+      try {
+        for (const alias of config.data.aliases[fullPath]) {
+          const aliasPath = path.posix.resolve(path.dirname(fullPath), alias).toLowerCase();
+          const aliasDir = path.dirname(aliasPath);
+          if (!(await mutableFilesystem.promises.exists(aliasDir))) {
+            await mutableFilesystem.promises.mkdir(aliasDir, {
+              recursive: true
+            });
+          }
+          if (!(await mutableFilesystem.promises.exists(aliasPath))) {
+            await mutableFilesystem.promises.symlink(fullPath, aliasPath);
+          } else {
+            console.warn(
+              `Alias '${aliasPath}' already exists. Is there a duplicate alias, or a page with the same name but different file extensions?`
```

Comment:
Adding some log tagging like [Mosaic][< error category>] would help with monitoring/alerting.

Reply:
Let's create a ticket and document what we want